crates/swc_ecma_parser/tests/typescript/issue-896/input.ts

```diff
@@ -1,2 +0,0 @@
-// @ts-ignore
-export type * from '../typings'
```

Comment:
tsc doesn't throw error for this with `@ts-ignore`. However, tsc doesn't throw error for almost other errors with `@ts-ignore` too. It's not that this error is special.